src/hooks/usePhaseData.ts

```diff
@@ -23,11 +21,11 @@ type GetPhaseData = {
 };
 
 export type ActivityData = {
-  name: string;
+  translatedShortName?: string;
+  translatedFullName: string;
```

Comment:
Doesn't this need to be optional too?

```suggestion
  translatedFullName?: string;
```